### PR TITLE
Updated variable names

### PR DESCRIPTION
**Pull request by donaldchen34**
Description of PR here...

Issue: Closes #

## Changes
-Updated variable names:
I forgot to edit some of the variable names after the initial change
-Also added the library install commands into the README
### Frontend:
- 
### Backend:
-

### CODE REVIEW

**Review by svhnstorage — COMMENTED** [review]
Did these fixes patch the issue of only 1 response being returned?

**Comment by donaldchen34** [comment]
> u

yeaa hopefully. So basically before I used the dicts to check if shapes were in our list and if it was, it would be converted to its number representation. After changing the shape dict to set, i forgot to change the condition check in the functions.

File: backend/chatbot.py [hunk 1]
```diff
@@ -45,8 +45,8 @@ def checkAction(action):
     return action in ACTIONS
 
 def checkLocation(row,col):
-    return row > GRID_SIZE or col > GRID_SIZE \
-        or row < 0 or col < 0
+    return not (row > GRID_SIZE or col > GRID_SIZE \
```
Comment (JasperCheung) [inline backend/chatbot.py]: shouldn't it be GRID_SIZE - 1? if ur grid size is 3 ur highest index is 2.

Comment (donaldchen34) [reply backend/chatbot.py]: It should be

**Review by JasperCheung — APPROVED** [review]
nice

File: backend/chatbot.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 nlp = spacy.load('en_core_web_sm')
 
-ACTIONS = {'FIND':findShape, 'DELETE':delShape, 'ADD':addShape, 'MOVE':moveShape, 'PICK':holdShape}
+ACTIONS = {'FIND':findShape, 'DELETE':delShape, 'ADD':addShape, 'MOVE':moveShape, 'HOLD':holdShape}
```
Comment (joshcwinton) [inline backend/chatbot.py:8]: nice

File: backend/chatbot.py [hunk 3]
```diff
@@ -45,8 +45,8 @@ def checkAction(action):
     return action in ACTIONS
 
 def checkLocation(row,col):
-    return row > GRID_SIZE or col > GRID_SIZE \
-        or row < 0 or col < 0
+    return not (row > GRID_SIZE - 1 or col > GRID_SIZE -1  \
```
Comment (joshcwinton) [inline backend/chatbot.py:48]: makes sense

File: backend/chatbot.py [hunk 4]
```diff
@@ -55,9 +55,15 @@ def emptyPosition(row,col):
 #Returns the reponse_number based on what conditions are met
 def doAction(action,shape,color,row,col):
 
-    if action == 'add':
+    if not checkShape(shape) == 1: # Check if shape is valid
+        return 2
```
Comment (joshcwinton) [inline backend/chatbot.py:59]: could we move to using a keyword or a direct function call here?

**Review by joshcwinton — APPROVED** [review]

**Pull request merged**